cpg-language-cxx/src/test/resources/comments.c

```diff
@@ -0,0 +1,20 @@
+/**
+ * Documentation comment
+ */
+void functionDoc() {
+  return;
+}
+
+/*
+ * Multi-line
+ * comment
+ */
+void functionMultiline() {
+  return;
+}
+
+// Simple comment
+
```

Comment:
Is this newline in between intentional?

Reply:
Yes, it is. The comment matcher should still identify it even with the additional newline between the comment and the function